.github/workflows/trigger_release.yml

```diff
@@ -50,10 +50,7 @@ jobs:
           node-version: 18
           check-latest: true
 
-      - uses: actions/checkout@v4
-        with:
-          fetch-depth: 25
-          persist-credentials: false # to allow start-release to override the git credentials
+      - run: git clone https://github.com/vercel/next.js.git --depth=25 .
```

Comment:
We need to ensure we're cloning the correct ref like `actions/checkout` does to ensure backport handling works